app/build.gradle

```diff
@@ -65,5 +64,5 @@ dependencies {
     implementation 'com.github.felHR85:UsbSerial:6.0.6'
     implementation 'com.github.AppIntro:AppIntro:4.2.3'
     implementation 'org.jfree:jfreechart:1.0.14'
-    implementation 'com.opencsv:opencsv:4.6'
+    implementation 'com.opencsv:opencsv:4.6', { exclude group: "commons-logging", module: "commons-logging" }
```

Comment:
What was the issue with this library? If it's making too much errors we can remove it. Because as far as I remember we don't use any parsing library in PSLab to handle CSV files :thinking:

Reply:
It has a particular module of `commons-logging` (consisting of logging implementations) that was basically causing conflicts with the same functionality now offered by Android(after the updates specifically). Yes, you are correct, the library is quite old and it seems they haven't kept the required changes with the pace of AS development. I think it should probably work fine now but not sure since I haven't meticulously researched on it 😕